packages/docker/src/utils.ts

```diff
@@ -41,7 +41,8 @@ export function optionsWithDockerEnvs(
     'DOCKER_HOST',
     'DOCKER_STACK_ORCHESTRATOR',
     'DOCKER_TLS_VERIFY',
-    'BUILDKIT_PROGRESS'
+    'BUILDKIT_PROGRESS',
+    'PATH'
```

Comment:
I would move this as a different check since dockerCliEnvs is specific to the docker, and PATH is more related to the supported tooling :relaxed: 

Can you add it to newOptions? 
```js
// ...

// Set docker envs or overwrite provided ones
for (const [key, value] of Object.entries(dockerEnvs)) {
    newOptions.env[key] = value as string
}

newOptions.env["PATH"] = process.env.PATH
```

Reply:
made that change - thanks